app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/model/VpnDatabaseModels.kt

```diff
@@ -25,20 +25,21 @@ import com.duckduckgo.mobile.android.vpn.trackers.AppTrackerEntity
 
 @Entity(
     tableName = "vpn_tracker",
-    indices = [Index(value = ["timestamp"])]
+    indices = [Index(value = ["bucket"])],
+    primaryKeys = ["bucket", "domain", "packageId"]
```

Comment:
nit: use trackerCompanyId (Int) instead of domain (text) as primary key

Reply:
As discussed, we company ID would change the semantics. As multiple tracker domains can belong to the same company ID (eg. Google).
And we need to search per tracker domain